src/formats/chips.hpp

```diff
@@ -159,8 +159,8 @@ class CHIPSProcessor : virtual public PacketProcessor {
 	    typedef aligned256_type otype;
 	    otype* __restrict__ aligned_data = (otype*)data;
 	    for( int t=0; t<nseq; ++t ) {
-		    for( int c=0; c<nchan; ++c ) {
-			    ::memset(&aligned_data[src + nsrc*(c + nchan*t)],
+		    for( int c=0; c<128; ++c ) {  
+			    ::memset(&aligned_data[src + nsrc*(c + 128*t)],
```

Comment:
I think these should still be `nchan` in the missing source blanker.

Reply:
@HaraKrish will test leaving it at `nchan` (and above on line 145)

Reply:
> @HaraKrish will test leaving it at `nchan` (and above on line 145)

This is done